### PR TITLE
WIP Fixing ESLint issues

### PR DESCRIPTION
**Pull request by Unesdala**

### CODE REVIEW

**Comment by Unesdala** [comment]
@JoshuaVSherman - There's 4 anys left that I can't quite figure out.

If you want to either fix them up yourself if you have time, or wait till Tuesday and do it in a live coding session, I'd appreciate it ^^

**Comment by JoshuaVSherman** [comment]
@Unesdala also there are two codeclimate issue to fix before assigning me to review please

**Comment by JoshuaVSherman** [comment]
@Unesdala also, when a merge is ready for review, please remove the `WIP` from its title

File: src/containers/Family/FamilyContent.tsx [hunk 1]
```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import ReactHtmlParser from 'react-html-parser';
 import PicSlider from '../../components/PicSlider';
 import ELCALogo from '../../components/elcaLogo';
+import { Store } from '../../redux/mapStoreToProps';
 
 const pageName = 'family';// eslint-disable-next-line max-len
 const additionalContent = '<hr><h5>Resources for Families</h5><p><strong><em>Roots and Wings</em></strong>, a monthly publication of the Virginia Synod that provides suggestions of daily family time activities:<a target="_blank" href="https://www.vasynod.org/ministries/roots-and-wings/">https://www.vasynod.org/ministries/roots-and-wings/</a></p><p><strong>Ideas to bring worship to the home:</strong>&nbsp;<a target="_blank" href="https://www.vasynod.org/wp-content/uploads/2012/11/Worship-in-the-Home.pdf">https://www.vasynod.org/wp-content/uploads/2012/11/Worship-in-the-Home.pdf</a></p><p><strong>Hand Prayer:</strong>&nbsp;<a target="_blank" href="http://www.vibrantfaithathome.org/item/hand-prayers">http://www.vibrantfaithathome.org/item/hand-prayers</a></p>';
-const FamilyContent = ({ familyPics }: any) => (
+const FamilyContent = ({ familyPics }: Store['familyPics']): JSX.Element => (
```
Comment (JoshuaVSherman) [inline src/containers/Family/FamilyContent.tsx:9]: @Unesdala this type is incorrect for familyPics prop, please fix

File: src/containers/Family/index.tsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 import DefaultFamilyContent from './FamilyContent';
-import mapStoreToProps from '../../redux/mapStoreToProps';
+import mapStoreToProps, { Store } from '../../redux/mapStoreToProps';
 import commonUtils from '../../lib/commonUtils';
 
-export const Family = ({ familyPics }: any) => {
-  commonUtils.setTitleAndScroll('Children & Families', window.screen.width);
+const pageName = ('Children & Families');
+
+export const Family = ({ familyPics }: Store['familyPics']): JSX.Element => {
```
Comment (JoshuaVSherman) [inline src/containers/Family/index.tsx:9]: @Unesdala this type is incorrect for familyPics prop, please fix

File: src/containers/Youth/index.tsx [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import DefaultYouthContent from './YouthContent';
-import mapStoreToProps from '../../redux/mapStoreToProps';
+import mapStoreToProps, { Store } from '../../redux/mapStoreToProps';
 import commonUtils from '../../lib/commonUtils';
 
-export const Youth = ({ youthPics }: any) => {
+export const Youth = ({ youthPics }: Store['youthPics']): JSX.Element => {
```
[the remark this answers is not in the document]
Comment (JoshuaVSherman) [reply src/containers/Youth/index.tsx:7]: @Unesdala this is still incorrect here as well, the youthPics prop, please fix

**Comment by Unesdala** [comment]
@JoshuaVSherman Since the main changes have been split into the other 2 PRs, closed this PR.

**Pull request closed without merging**